src/commands/publish.ts

```diff
@@ -426,9 +426,17 @@ export async function publishMain(argv: PublishOptions): Promise<any> {
   logger.debug('Revision to publish: ', revision);
 
   const statusProvider = getStatusProviderFromConfig();
-  logger.info(`Using "${statusProvider.constructor.name}" for status checks`);
+  const statusProviderName = statusProvider.constructor.name;
   const artifactProvider = getArtifactProviderFromConfig();
-  logger.info(`Using "${artifactProvider.constructor.name}" for artifacts`);
+  const artifactProviderName = artifactProvider.constructor.name;
+  logger.info(' ');
+  logger.info(`Using "${statusProviderName}" for status checks`);
+  logger.info(`Using "${artifactProviderName}" for artifacts`);
+  if (statusProviderName !== 'zeus' && artifactProviderName !== 'zeus') {
+    logger.info(' ');
+    logger.info("    Zeus? Who's Zeus?");
```

Comment:
Haha
You sure you want to display that when providers are _not_ zeus, though?

Reply:
That was kind of the little joke. We're not reliant on Zeus anymore, so we can pretend we’ve never heard of him. But if it takes explaining, it's not funny. (This is what happens when nerds try to make jokes.) Can take it out.